gitgud/skills/newbasics/__init__.py

```diff
@@ -0,0 +1,44 @@
+from gitgud.skills.level_builder import BasicLevel
+from gitgud.skills.util import Skill
+from gitgud import operations
+from gitgud.skills.user_messages import firstcommit_status
+
+
+class FirstCommit(BasicLevel):
+    def _setup(self):
+        file_operator = operations.get_operator()
+        file_operator.destroy_repo()
+        file_operator.use_repo()
+
+    def status(self):
+        firstcommit_status(*self._get_state())
+
+    def _get_state(self):
+        created = False
+        added = False
+        committed = True
+        file_operator = operations.get_operator()
+        created_files = file_operator.get_working_directory_content()
+        if created_files:
+            created = True
+        added_files = file_operator.get_staging_content()
+        if added_files:
+            added = True
+        try:
+            file_operator.repo.head.commit
+        except ValueError:
+            committed = False
+
+        return created, added, committed
```

Comment:
What do you think about doing `return bool(created_files), bool(added_files), committed`?

Reply:
See [this comment](https://github.com/benthayer/git-gud/pull/305#discussion_r486033022).